github/util.py

```diff
@@ -221,3 +224,95 @@ def retrieve_email(username: str):
         )
     )
 
+def _create_team(
+    github: GitHub,
+    organization_name: str,
+    team_name: str
+):
+    # passed GitHub object must have org. admin authorization to create a team
+    organization = github.organization(organization_name)
+    team = _retrieve_team_by_name(organization, team_name)
+    if team:
+        util.verbose("Team {name} already exists".format(name=team_name))
+        return
+
+    try:
+        organization.create_team(name=team_name)
+        util.info("Team {name} created".format(name=team_name))
+    except ForbiddenError as err:
+        util.fail("{err} Cannot create team {name} in org {org} due to missing privileges".format(
+            err=err,
+            name=team_name,
+            org=organization_name
+        ))
+
```

Comment:
nit: two lines between functions (methods are separated by one line) according to pep-8